src/main/java/uk/gov/hmcts/reform/hmc/api/services/PrlUpdateServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.hmc.api.services;
+
+import java.util.Arrays;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.HttpServerErrorException;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.hmc.api.exceptions.PrlUpdateException;
+import uk.gov.hmcts.reform.hmc.api.model.request.Hearing;
+
+@Service
+public class PrlUpdateServiceImpl implements PrlUpdateService {
+
+    @Autowired private AuthTokenGenerator authTokenGenerator;
+
+    private static final Logger LOG = LoggerFactory.getLogger(PrlUpdateServiceImpl.class);
+
+    @Autowired PrlUpdateApi prlUpdateApi;
+
+    /**
+     * This method will update the Prl with hearing data.
+     *
+     * @param hearing data to update prl.
+     * @return isPrlRespSuccess, Boolean - prlHearing update.
+     */
+    @Override
+    @SuppressWarnings("unused")
+    public Boolean updatePrlServiceWithHearing(Hearing hearing) {
+        Boolean isPrlRespSuccess = false;
+        LOG.info("updatePrlServiceWithHearing");
+
+        if ("BBA3".equals(hearing.getHmctsServiceCode())) {
+
+            try {
+                ResponseEntity responseEntity =
+                        prlUpdateApi.prlUpdate(
+                                Arrays.asList(getServiceAuthorisationToken()).toString(), hearing);
+                LOG.info("PRL call completed successfully" + responseEntity.getStatusCode());
+                isPrlRespSuccess = true;
```

Comment:
No need to check 200 explicitly. 2XX will be handled with in the TRY block, 4XX(Client Exception) and 5XX(Server Exception) will be handled in the CATCH block. 3XX is not applicable, because it is not in the destination app.For Reference please see the below link 
https://github.com/hmcts/ccpay-service-request-cpo-update-service/blob/master/src/main/java/uk/gov/hmcts/reform/services/CpoUpdateServiceImpl.java